Update master

agahkarakuzu opened the pull request:
Pull the latest upstream.

Gilles86 commented on src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:
```diff
@@ -136,6 +136,56 @@ modalities include:
+| T2 weighted images                         | T2w     | Conventional | Denotes images with predominant T2 contribution.                                                                                                                                                                                |
+| Proton density weighted images             | PDw     | Conventional | Denotes images with predominant proton density (PD) contribution.                                                                                                                                                               |
+| T2 star weighted images                    | T2starw | Conventional | Denotes images with predominant T2* contribution, typically images acquired using a GRE sequence with low flip angle, long echo time and long repetition time. Please note that this suffix is not a surrogate for `T2starmap`. |
+| Fluid Attenuated Inversion Recovery Images | FLAIR   | Conventional | Denotes images with predominant T2 contribution (a.k.a T2-FLAIR), in which signal from fluids (e.g. CSF) is nulled out by adjusting inversion time, coupled with notably long repetition and echo times.              |
```
Just want to point out that the suffix FLAIR is maybe not what we would have come up with, because, strictly speaking, it is not a well-defined weighting. But since everyone in fMRI would use this interchangeably with T2-FLAIR and we want backwards-compatibility, we will leave it in like this. This was also discussed in multiple BIDS meetings.

Gilles86 commented on src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:
```diff
@@ -136,6 +136,56 @@ modalities include:
+| Proton density weighted images             | PDw     | Conventional | Denotes images with predominant proton density (PD) contribution.                                                                                                                                                               |
+| T2 star weighted images                    | T2starw | Conventional | Denotes images with predominant T2* contribution, typically images acquired using a GRE sequence with low flip angle, long echo time and long repetition time. Please note that this suffix is not a surrogate for `T2starmap`. |
+| Fluid Attenuated Inversion Recovery Images | FLAIR   | Conventional | Denotes images with predominant T2 contribution (a.k.a T2-FLAIR), in which signal from fluids (e.g. CSF) is nulled out by adjusting inversion time, coupled with notably long repetition and echo times.              |
+| Inplane T1                                 | inplaneT1 | Conventional | T1-weighted anatomical image matched to functional acquisition                                                           |
```
This one I really don't like. It's ambivalent and superfluous. Ambivalent, because, strictly speaking, it is not clear whether this is a T1-weighted image or a T1-map. Superfluous, because the orientation matrix and matrix size of the nifti-file can be use to deduce which part of the brain is covered and the `acq-`-key/value pair can be used to indicate the intended use-case. 

Then again, we don't want to pick a fight.
What do you think, @agahkarakuzu , @KirstieJane ?

agahkarakuzu replied:
We discussed with this @KirstieJane, looks like it is quite commonly used (low res image to improve registration), so removing it may stir the pot.

Gilles86 commented on src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md:
```diff
@@ -136,6 +136,56 @@ modalities include:
+| T2 star weighted images                    | T2starw | Conventional | Denotes images with predominant T2* contribution, typically images acquired using a GRE sequence with low flip angle, long echo time and long repetition time. Please note that this suffix is not a surrogate for `T2starmap`. |
+| Fluid Attenuated Inversion Recovery Images | FLAIR   | Conventional | Denotes images with predominant T2 contribution (a.k.a T2-FLAIR), in which signal from fluids (e.g. CSF) is nulled out by adjusting inversion time, coupled with notably long repetition and echo times.              |
+| Inplane T1                                 | inplaneT1 | Conventional | T1-weighted anatomical image matched to functional acquisition                                                           |
+| Inplane T2                                 | inplaneT2 | Conventional | T2-weighted anatomical image matched to functional acquisition                                                           |
```
Same issue as inplaneT1

agahkarakuzu commented:
e5420db

effigies committed:
[INFRA] Relax line length limit for linting YAML files (#673)

* MNT: Relax line length limit in YAML

* MNT: Ignore pycache in tools dir

* CI: Update configuration based on Travis validator

Also bump Python version and cache pip

tsalo committed:
[SCHEMA] Add derivatives entities to the schema (#645)

* Add derivatives entities.

* Apply suggestions from code review

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Differentiate derivative entities from raw entities.

* Update entities page.

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

tsalo committed:
[ENH] Add part entity for complex-valued data (#424)

* Add part entity.

* Add self to contributors file.

* Address @yarikoptic's review.

* Fix entity table spacing.

* Add missing OPTIONAL for part under func (bold phase).

* Fix spacing again.

* Remove OPTIONAL under func (bold phase). Wasn't missing.

* Fix entity order (part goes after run).

* Fix typo.

* Incorporate part into schema.

* Update definition and examples to match BEP001.

* Fix code snippets.

* Fix filename patterns now that real/imag is supported.

* Apply suggestions from code review

- Change radians to rad.
- Treat `part` like `run`. Can only be omitted if only magnitude image is provided.

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Support part with BOLD and update entity table.

* Update src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Update entity table and entity definitions.

* Add link in text.

* Remove section header.

* Address review.

* Update src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

* Deprecate phase suffix.

* Fix link.

* Fix link again.

* Include deprecation in table.

* Fix table.

* Update 01-magnetic-resonance-imaging-data.md

* Apply suggestions from code review

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Add entity field.

* Fix bug and generate appendix.

* Fix table.

* Update src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Improve part definition.

Co-authored-by: Chris Markiewicz <effigies@gmail.com>
Co-authored-by: Chris Markiewicz <markiewicz@stanford.edu>

agahkarakuzu committed:
[ENH] BEP001 - New entity: flip (#672)

* [ENH] Add flip entity

- Modify `echo` entity respectively

* revert entities.md and update the schema

* Use schema tools

* [FIX] Format field

* Update src/schema/entities.yaml

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Update src/schema/entities.yaml

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Wording suggestion by @tsalo

* [ENH] Add flip entity

- Modify `echo` entity respectively

* revert entities.md and update the schema

* Use schema tools

* [FIX] Format field

* Update src/schema/entities.yaml

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Update src/schema/entities.yaml

Co-authored-by: Chris Markiewicz <effigies@gmail.com>

* Wording suggestion by @tsalo

Co-authored-by: Chris Markiewicz <effigies@gmail.com>
